README.md

```diff
@@ -1,55 +1,231 @@
+>
+> ```sh
+> docker-compose up -d
+> ```
+
+#### Using `locally build`
+
+> ![docker-compose](https://img.shields.io/badge/Docker-2496ED.svg?style=flat&logo=Docker&logoColor=white)
+>
+> ```sh
+> docker-compose up -d --build
+> ```
+
+---
 
-```bash
-docker-compose up postgres -d
-```
+### ⚙️ Local development
 
-After [node](https://github.com/nodesource/distributions#installation-instructions) is installed, enable `corepack` to use `yarn`: [yarn](https://yarnpkg.com/getting-started/install)
+One-time setup:
 
-```bash
-sudo corepack enable
-```
+> Enable `corepack` to use [`yarn`](https://yarnpkg.com/getting-started/install)
+>
+> ```sh
+> sudo corepack enable
+> ```
+
+Starting development:
 
-Install all dependencies:
+> Start the database
+>
+> ```sh
+> docker-compose up postgres -d
+> ```
+>
+> Install all dependencies
+>
+> ```sh
+> yarn install
+> ```
+>
+> Apply migrations (WARNING: this could drop all of your local data, pay attention)
+>
+> ```sh
+> yarn prisma migrate dev
+> ```
+>
+> Start the application
+>
+> ```sh
+> yarn dev
+> ```
+>
+> All ui changes are automatically reloaded, but you need to restart the server when changing the api
 
-```bash
-yarn install
-```
+Developing changes that include database schema changes:
 
-Before starting the application, you need to run the migrations (WARNING: this could drop all of your local data, pay
-attention)
+> Push your changes without creating a migration
+>
+> ```sh
+> yarn prisma db push
+> ```
+>
+> When you are done with your changes, create a migration
+>
+> ```sh
+> yarn prisma migrate dev
+> ```
+>
+> All other help can be found in the [Prisma Documentation](https://www.prisma.io/docs/orm/prisma-migrate)
 
-```bash
-yarn migrate
-```
+---
 
-Start the application:
+## 📒 Changelog
 
-```bash
-yarn dev
-```
+All changes to the project are documented in
+the [Changelog](https://github.com/jo-gross/Cocktail-Manager/blob/main/docs/CHANGELOG.md), automatically generated by
+when creating a new release.
 
-## Docker
+---
+
+## 🤝 Contributing & Support
+
+To grow the project, we need your help! See the links below to get started.
 
-Build image
+- [🔰 Contributing Guide][1]
+- [👋 Start a Discussion][2]
+- [🐛 Open an Issue][3]
 
-```bash
-docker-compose -f docker-compose.yaml build
-```
+[1]: https://github.com/jo-gross/cocktail-manager/blob/main/CONTRIBUTING.md '🔰 Contributing Guide'
 
-Push to local registry
+[2]: https://github.com/jo-gross/cocktail-manager/discussions '👋 Start a Discussion'
 
-```bash
-docker push 10.200.16.1:8881/cocktails/cocktail-recipe-cocktails:latest
-```
+[3]: https://github.com/jo-gross/cocktail-manager/issues '🐛 Open an Issue'
+
+<p align="left">
+  <a href="https://github.com/jo-gross/cocktail-manager/graphs/contributors">
+    <img src="https://contrib.rocks/image?repo=jo-gross/cocktail-manager" />
+  </a>
+</p>
+
+As some asked, you´re also welcome to support financial to reduce costs, like hosting and domain fees, by donating.
```

Comment:
Is there already some hosting and domains?

Reply:
It´s in progress but not released yet. I can remove that part until its public available if you mind.

Reply:
Currently hanging on legal stuff like privacy terms and terms of services (better save than sorry if i loose all of your fine developed recipes ;D)

Reply:
I would remove it until there is something or just add the Github Sponsor button?

Reply:
Alright